week-2/implement/lookup.js

```diff
@@ -1,17 +1,13 @@
-function createLookup() {
+function createLookup(countryAndCurrencyArray) {
+  return countryAndCurrencyArray.reduce((obj, [countryCode, currency]) => {
+    obj[countryCode] = currency;
+    return obj;
+  }, {});
   // implementation here
 }
 
```

Comment:
Great work!

Reply:
You may add a check if array passed is empty or the inner array is empty!

Reply:
Thanks for the orientation. I will do this until tomorrow (friday 05/01). Thanks.

Reply:
Check added and new tests made.